### PR TITLE
Enable site admins to create member pages again

### PR DESCRIPTION
**Pull request by Genyus**
Fixes https://codingblackfemales.atlassian.net/browse/TECH-210

### CODE REVIEW

File: web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   * ...and/or other possibilities.
+   *
+   * Ideally, it is not considered best practise to attach more than a
+   * single DOM-ready or window-load handler for a particular page.
+   * Although scripts in the WordPress core, Plugins and Themes may be
+   * practising this, we should strive to set a better example in our own work.
+   */
+})(jQuery);
+/******/ })()
+;
```
Comment (tanyapowell) [inline web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.js:36]: Missing empty line at end of this file

Comment (Genyus) [reply web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.js:36]: It’s a generated file. Will look at adding a lint step after building, but don’t want to hold up this PR for that.

File: web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.min.asset.php [hunk 2]
```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'a10c07783c2ec2f891c1');
```
[the remark this answers is not in the document]
Comment (Genyus) [reply web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.min.asset.php:1]: Webpack, IIRC

File: web/app/themes/cbf-academy/functions.php [hunk 3]
```diff
@@ -72,3 +72,15 @@ function cbf_academy_bp_core_get_table_prefix( $base_prefix ) {
...
+ * Return true as 'buddypages-member-pages' option value for site admins
+ *
+ * @param bool   $value The value of the 'buddypages-member-pages' site option
+ * @param string $option The option name
+ * @return bool
+ */
+function cbf_academy_option_buddypages_member_pages( $value, $option ) {
+	return ! is_admin() && current_user_can( 'manage_options' ) || $value;
+}
+add_filter( 'option_buddypages-member-pages', 'cbf_academy_option_buddypages_member_pages', 10, 2 );
```
[the remark this answers is not in the document]
Comment (Genyus) [reply web/app/themes/cbf-academy/functions.php:86]: That’s correct

**Review by tanyapowell — APPROVED** [review]

**Pull request merged**